tools/coverity.sh

```diff
@@ -135,7 +134,6 @@ _upload()
 	           --silent --write-out "\n%{http_code}\n" \
 	           --form project=$COVERITY_SCAN_PROJECT_NAME \
 	           --form token=$COVERITY_SCAN_TOKEN \
-	           --form email=$COVERITY_SCAN_NOTIFICATION_EMAIL \
```

Comment:
Judging by "Email can't be blank", email should always be set. It would probably make sense to ask Coverity to relax this.